src/everest/simulator/everest_to_ert.py

```
@@ -389,21 +389,31 @@ def _extract_seed(ever_config: EverestConfig, ert_config: dict[str, Any]) -> Non
 
 
 def _extract_results(ever_config: EverestConfig, ert_config: dict[str, Any]) -> None:
-    objectives_names = [objective.name for objective in ever_config.objective_functions]
-    constraint_names = [
-        constraint.name for constraint in ever_config.output_constraints
-    ]
+    everest_objectives = []
+    for objective in ever_config.objective_functions:
+        everest_objectives.append(
+            {"name": objective.name, "input_file": objective.name}
+        )
+    if everest_objectives:
```

Suggest doing a validation for constraints / objectives:
If there are forward model results with `type: objective` and no corresponding objectives in the `objective_functions` section, we should raise an error imo. Same for constraints.

We COULD validate the other way (but that would be a breaking change and may be deferred to later)

---

This sounds like a config validation and not something that should be done here. I will add an issue for this.

---

https://github.com/equinor/ert/issues/10896

---

Can you rename the issue to something more specific?